aspnetcore/mvc/views/razor.md

```diff
@@ -568,6 +568,17 @@ The code renders the following HTML:
 
 ::: moniker-end
 
+### @attribute
+
+The `@attribute` directive sets the given attribute on the generated view.
+
+```cshtml
+@attribute [Serializable]
+```
+
+> [!WARNING]
+> In preview6 there is a known issue where @attribute directives do not work in `_Imports.razor` and `_ViewImports.cshtml`. This will be fixed in preview7.
```

Comment:
```suggestion
> In 3.0.0-preview6 there is a known issue where `@attribute` directives do not work in `_Imports.razor` and `_ViewImports.cshtml`. This will be fixed in 3.0.0-preview7.
```